package.json

```diff
@@ -23,7 +24,6 @@
     "webpack": "^2.6.1"
   },
   "dependencies": {
-    "react": "^15.6.0",
-    "react-dom": "^15.6.0"
+    "react-lite": "0.15.30"
```

Comment:
I chose this version to match what the cockpit main repo [is using](https://github.com/cockpit-project/cockpit/blob/ea5507dfd4cd167ea9a38c6ec31bc08a26387797/package.json#L26). I don't think the exact version matters so much, since we only use basic stuff.